runtimes/js/encore.dev/api/mod.ts

```diff
@@ -261,5 +265,133 @@ api.static = function staticAssets(options: StaticOptions) {
   return new StaticAssets(options);
 };
 
+export interface MiddlewareOptions {
+  /**
+   * Configuration for what endpoints that should be targeted by the middleware
+   */
+  target?: {
+    /**
```

Comment:
are we happy with naming here (eg isRaw vs exposed, use a more unified naming)? 
anything that we should remove or add?

Reply:
and then I guess `expose` and not `exposed` as that is what it is called in the APIOptions?

Reply:
Yeah, sounds good